tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/action/start/StartActionTest.java

```diff
@@ -58,7 +58,6 @@ public void testStartActionNegative() {
         BAssertUtil.validateError(result, indx++, "'wait' cannot be used with actions", 72, 18);
         BAssertUtil.validateError(result, indx++, "action invocation as an expression not allowed here", 72, 28);
         BAssertUtil.validateError(result, indx++, "action invocation as an expression not allowed here", 76, 25);
-        BAssertUtil.validateError(result, indx++, "incompatible types: expected 'other', found 'int'", 90, 13);
```

Comment:
Do we have test cases where an error with `other` is the only error and isn't suppressed?

Reply:
In `tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/lambda/IterableOperationsTests.java`

Following diagnostic related to testcase 1 is preserved while diagnostics related to testcase2 is suppressed.

Testcase 1

BAssertUtil.validateError(negative, index++, "incompatible types: expected '[other,other]', found 'string[]'",
        48, 18);

Testcase 2

BAssertUtil.validateError(negative, index++,
                          "invalid list binding pattern: attempted to infer a list type, but found 'other'",
                          48, 18);